src/Arcus.WebApi.Unit/Security/Configuration/SecretProviderConfigurationBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Configuration;
+using Xunit;
+using Arcus.WebApi.Security.Configuration;
+using System.Collections.Generic;
+
+namespace Arcus.WebApi.Unit.Security.Configuration
+{
+    public class SecretProviderConfigurationBuilderExtensionsTests
```

Comment:
Can we add a test where the configurationKey is not in the vault?

Reply:
Yep, done.

Reply:
Thanks!